scripts/Interface/ui/sans_isis/beam_centre.py

```diff
@@ -7,22 +7,24 @@
 from __future__ import (absolute_import, division, print_function)
 
 from abc import ABCMeta, abstractmethod
-
-from PyQt4 import QtGui, QtCore
+from qtpy import QtGui, QtCore, QtWidgets
 from six import with_metaclass
-from . import ui_beam_centre
-from mantidqtpython import MantidQt
+
+from mantidqt.utils.qt import load_ui
+from mantidqt.widgets import messagedisplay
 from sans.gui_logic.gui_common import get_detector_from_gui_selection, \
-    get_detector_strings_for_gui, get_string_for_gui_from_reduction_mode
+     get_detector_strings_for_gui, get_string_for_gui_from_reduction_mode
 
 try:
     _fromUtf8 = QtCore.QString.fromUtf8
 except AttributeError:
     def _fromUtf8(s):
         return s
 
+Ui_BeamCentre, _ = load_ui(__file__, "beam_centre.ui")
```

Comment:
Same as above, why are the ui file's being set as global variables in the file instead of object instance variables?